src/main/java/org/mockito/session/MockitoSessionBuilder.java

```diff
@@ -30,17 +30,47 @@
      * Migrate from {@link MockitoAnnotations#initMocks(Object)}
      * to {@link MockitoSession}!
      * <p>
+     * This method may be called multiple times to add multiple, e.g. nested, test class instances.
+     * <p>
      * See code sample in {@link MockitoSession}.
      *
      * @param testClassInstance test class instance that contains fields with Mockito annotations to be initialized.
-     *  Passing {@code null} is permitted and will make the session use a default value.
-     *  The current default is '{@code new Object()}'.
+     *  Passing {@code null} is permitted but will be ignored.
      * @return the same builder instance for fluent configuration of {@code MockitoSession}.
      * @since 2.7.0
      */
     @Incubating
     MockitoSessionBuilder initMocks(Object testClassInstance);
 
+    /**
+     * Adds the test class instances for initialization of fields annotated with Mockito annotations
+     * like {@link org.mockito.Mock}.
+     *
+     * @param testClassInstances test class instances that contains fields with Mockito annotations to be initialized.
+     *  Passing {@code null} or an empty array is permitted but will be ignored.
+     * @return the same builder instance for fluent configuration of {@code MockitoSession}.
+     * @see #initMocks(Object)
+     * @since 2.13.4
+     */
+    @Incubating
+    MockitoSessionBuilder initMocks(Object... testClassInstances);
```

Comment:
I suggest to kill the other 'initMocks' method and just leave the vararg. It is a compatible change.

Reply:
AFAIK it's a source compatible change, but breaks binary compatibility because old code is compiled against a method with signature `initMocks(Object)` which would no longer exists, because there's now only `initMocks(Object[])`.

Reply:
Good idea.